corehq/apps/userreports/tests/test_report_data.py

```diff
@@ -20,7 +20,6 @@
 class ReportDataTest(TestCase):
 
     def setUp(self):
-        super(ReportDataTest, self).setUp()
```

Comment:
Was it intentional to remove the `super().setUp()` call entirely?

Reply:
Yeah my understanding is when inheriting from TestCase, it isn't necessary to call super since that is not at the `TestCase` level.

Reply:
That was my initial take as well, but I think by convention, we're still preferring including essentially a no-op `super().setUp()` so that you don't have to remember that `setUp()` doesn't need to call its parent class, while `setUpClass()` does.

Reply:
Ah ok that makes sense. Will revert those changes 👍🏻

Reply:
See 3b2a4d4e7aed4c5f4752ac1c515c60b49520966d (there were a couple of tests that I modified that had already not specified super().setUp(), so I added it in those cases)